stan/math/prim/prob/ordered_probit_lpmf.hpp

```diff
@@ -84,12 +86,13 @@ return_type_t<T_loc, T_cut> ordered_probit_lpmf(const T_y& y,
     int K = c_vec[i].size() + 1;
 
     if (y_vec[i] == 1) {
-      logp_n += log1m(Phi(lambda_vec[i] - c_vec[i].coeff(0)));
+      logp_n += log1m_exp(std_normal_lcdf(lambda_vec[i] - c_vec[i].coeff(0)));
```

Comment:
You could improve numerical stability even more by using `std_normal_lccdf` here ... except that `std_normal_lccdf` has much less robust implementation than `std_normal_lcdf` despite mathematically being the same function.
So I'm going to suggest instead
```suggestion
      logp_n += std_normal_lcdf(c_vec[i].coeff(0) - lambda_vec[i]);
```

Reply:
Clever! Have pushed the change